src/js/mapview/MapView.js

```diff
@@ -25,7 +25,7 @@ L.DNC.MapView = L.Class.extend({
     _setupMap : function () {
 
         L.mapbox.accessToken = 'pk.eyJ1Ijoic3ZtYXR0aGV3cyIsImEiOiJVMUlUR0xrIn0.NweS_AttjswtN5wRuWCSNA';
-        this._map = L.mapbox.map('map', 'svmatthews.hf8pfph5', {
+        this._map = L.mapbox.map('map', 'examples.map-zr0njcqy', {
```

Comment:
@svmatthews Noticed that my basemap now says "EXAMPLE" and comes with some points of interest around the DB area.  Is this intentional?
![screen shot 2015-04-29 at 9 43 34 pm](https://cloud.githubusercontent.com/assets/897290/7406025/0c87c05e-eeb9-11e4-886b-a0217eb2d70b.png)